Functions/GameOfLife.cs

```diff
@@ -2,7 +2,49 @@
 
 public class GameOfLife
 {
-	/*
-	 * TODO: Write helper function for 'Game of life'
-	 */
+    public static int[,] UpdatedBoard(int[,] board, int rows, int cols)
+    {
+        int[,] newBoard = new int[rows, cols];
+        for (int row = 0; row < rows; row++)
+        {
+            for (int col = 0; col < cols; col++)
+            {
+                int current = board[row, col];
+                int amountOfNeighbors = AmountOfNeighbors(board, row, col, rows, cols);
+
+                if (amountOfNeighbors + current == 3 || amountOfNeighbors == 3)
+                    newBoard[row, col] = 1;
+                else
+                    newBoard[row, col] = 0;
+            }
+        }
+        return newBoard;
+    }
+
+    public static int AmountOfNeighbors(int[,] board, int row, int col, int rows, int cols)
```

Comment:
This logic could be simplified.
It is too complicated in my opinion for what it's supposed to do.
And there are no comments when this is not trivial!

Reply:
I added comments to this part. 
I solved it in this way in order to not add too many `if`s (it's not really complicated, but I do understand it's not trivial) 
Do you think I should change the logic anyways?